userspace/falco/configuration.cpp

```diff
@@ -167,6 +167,18 @@ void falco_configuration::merge_configs_files(const std::string& config_name, st
 	}
 }
 
+void falco_configuration::init_logger()
```

Comment:
This is somewhat a fix: initialize the logger before loading anything else from the config; this allows us to log at the correct level in case we need to.
NOTE: this wasn't a bug since no logging was present in the yaml loading phase in configuration.cpp; but now we have to log a warning when both new and old keys are defined.

All in all, logging is quite always the first thing that needs to be inited, thus the change.